### PR TITLE
feat: support users and users repos

### LINKED ISSUE #73: Add routes for users
Hello,

Could we add a route to have data about a user and his repos (same as for orgs).

### PR DESCRIPTION
**Pull request by Barbapapazes**
### 🔗 Linked issue

fix #73

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme, or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [x] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

This PR add a route to retrieve user's infos and user's repos.

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: routes/users/[name]/index.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import type { GithubUser } from "~types";
+
+export default eventHandler(async (event) => {
+  const name = getRouterParam(event, "name");
+  const user = await ghFetch(`users/${name}`);
+
+  return {
+    user: <GithubUser>{
+      id: user.id,
+      username: user.login,
```
Comment (pi0) [inline routes/users/[name]/index.ts:10]: We can additionally expose `name`, `twitter_username` as `twitter`, and `avatar`

File: README.md [hunk 2]
```diff
@@ -263,6 +263,47 @@ Multiple items can be separated by either `,` or `+` or ` ` (space). Each item c
...
+
+Find one github user by username.
+
+**Example:** https://ungh.cc/users/pi0
+
+```json
+{
+  "user": {
+    "id": 5158436,
+    "username": "pi0"
```
Comment (pi0) [inline README.md:276]: Can we please update docs too? 🙏🏼

Comment (Barbapapazes) [reply README.md:276]: Oh yes! I miss it.

**Review by pi0 — APPROVED** [review]
Thanks ❤️

**Pull request merged**